modules/openapi-generator/src/main/resources/ruby-client/api_client_typhoeus_partial.mustache

```diff
@@ -49,6 +49,7 @@
       header_params = @default_headers.merge(opts[:header_params] || {})
       query_params = opts[:query_params] || {}
       form_params = opts[:form_params] || {}
+      follow_location = opts[:follow_location] || false
```

Comment:
@moorec22 thanks for the PR. Overall it looks good to me.

One suggestion is defaulting `follow_location` to true as I think that's what the user of the Ruby SDK expected.

Reply:
oh that could work! @wing328 my only fear is that this would technically change functionality. Maybe I'm being overly-cautious, but if a user were rescuing 302s and handling them in some way, they would no longer get 302s (unless they add `follow_location: false`).

Is that too paranoid? I'd be happy with `true` or `false` just checking.

Reply:
My take is to go with `follow_location: true` to start with so that we can find out more how people are working around 302 status code at the moment (if they later report issue with this change).

I'll change the default value in another commit after merging this PR.

Reply:
Fantastic thank you!